proteomic_aggregation_and_visualization/python/proteomic_aggregation_and_visualization.ipynb

```diff
@@ -0,0 +1,4004 @@
+{
```

Comment:
<p>Can you change the axes labels to be a little more understandable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>